src/cards/stats-card.js

```diff
@@ -137,19 +137,6 @@ const renderStatsCard = (stats = {}, options = { hide: [] }) => {
     },
   };
 
-  const longLocales = [
-    "cn",
-    "es",
-    "fr",
-    "pt-br",
-    "ru",
-    "uk-ua",
-    "id",
-    "my",
-    "pl",
-  ];
-  const isLongLocale = longLocales.includes(locale) === true;
```

Comment:
We actually do want this logic to be present, if we just assume every locale is as long as 100px and shift it by that amount it will look weirdly gaped for short locales. 

So i think we should adjust depending on if the locale is long or not.

Reply:
You are right. I've reverted the commit and added "de" and "nl" to longLocal. This should fix the problem for now.
I think the best solution in the future would be to make the shift dependend on the characters counted for each line and then figure out the padding.
If you agree with me I will create an issue for that so that someone can improve that 👍